### PR TITLE
Register and Login page responsive

### LINKED ISSUE #450: Responsiveness of the register page on all devices💡[FEATURE]:
Register page is not responsive for small devices like mobiles or tabs.

I would want to work on this issue through tailwind CSS & make it responsive.

@codewarnab  please this issue to me under gssoc'24

https://github.com/Innovateninjas/Paws-frontend/assets/97687618/64147c31-0842-4168-bd6f-86eb90a5e8c6

### PR DESCRIPTION
**Pull request by the-t3ch-wizard**
## Description
Removed some unnecessary code from UserRegistration page.
Made Register and Login page responsive.

Fixes #450 , #419 and #380 

## Related Issue
None

## Screenshots / GIFs (if applicable)
Screenshot 2024-06-05 144802
Screenshot 2024-06-05 144751

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have added/updated relevant documentation (if needed)
- [x] I have tested the changes locally and they function as expected
- [x] I have ensured my code follows the project's coding standards

## Additional Notes
None

### CODE REVIEW

**Comment by codewarnab** [comment]
> ## Description
> Removed some unnecessary code from UserRegistration page. Made Register and Login page responsive.
> 
> Fixes #450 , #419 and #380
> 
> ## Related Issue
> None
> 
> ## Screenshots / GIFs (if applicable)
> Screenshot 2024-06-05 144802 Screenshot 2024-06-05 144751
> 
> ## Checklist:
> * [x]  I have performed a self-review of my code
> * [x]  I have added/updated relevant documentation (if needed)
> * [x]  I have tested the changes locally and they function as expected
> * [x]  I have ensured my code follows the project's coding standards
> 
> ## Additional Notes
> None

hey you did good job are you contributing under gssoc ?

**Comment by codewarnab** [comment]
image
make the create account text little bit more smaller 
image
decrease this padding from all sides probabalu this will fit the create account text in one line which will look better

**Comment by the-t3ch-wizard** [comment]
I'm working on the conflicts and will inform as soon as possible.

**Comment by the-t3ch-wizard** [comment]
is the conflicts resolved?

**Comment by the-t3ch-wizard** [comment]
@codewarnab  Please review my PR.
I have improved the ui as requested.
and since you asked, no i'm not contributing under gssoc

File: src/pages/user/UserRegistration.jsx [hunk 1]
```diff
@@ -2,43 +2,43 @@ import React, { useEffect, useState } from "react";
...
-  const isLogin = location.pathname !== "/register";
+  let isLogin = location.pathname !== "/register";
 
   const customButtonStyle = {
     borderRadius: "20px",
     background: "linear-gradient(to bottom, #b3d9ff, #3399ff)",
+
+    padding: "16px 32px",
+    marginTop: "1rem",
+    fontSize: "18px",
```
Comment (codewarnab) [inline src/pages/user/UserRegistration.jsx:33]: its still has duplicate keys

**Review by codewarnab — CHANGES_REQUESTED** [review]

**Comment by codewarnab** [comment]
> image make the create account text little bit more smaller image decrease this padding from all sides probabalu this will fit the create account text in one line which will look better

also did you chanages this what i mentioned ?

File: src/pages/user/UserRegistration.jsx (hunk 1 above)
Comment (the-t3ch-wizard) [reply src/pages/user/UserRegistration.jsx]: I had removed the duplicate keys in my recent commit.

**Comment by the-t3ch-wizard** [comment]
> > image make the create account text little bit more smaller image decrease this padding from all sides probabalu this will fit the create account text in one line which will look better
> 
> also did you chanages this what i mentioned ?

Yes I have improved the ui as per requested
The mobile ui :
Screenshot 2024-06-06 214828
The tablet ui : 
Screenshot 2024-06-06 215750

File: src/pages/user/UserRegistration.jsx [hunk 2]
```diff
@@ -8,12 +8,6 @@ import { FaEye, FaEyeSlash } from 'react-icons/fa6';
 import i1 from '../user/imgs/white-dog-pastel-blue-background-3d_89917-269.jpg';
 import { login, registration } from "../../utils/Functions/userAuthService";
 import LoginTextLink from "../../Components/shared/LoginTextLink";
```
[the remark this answers is not in the document]
Comment (the-t3ch-wizard) [reply src/pages/user/UserRegistration.jsx:9]: I have added this feature too in my latest commit

**Comment by the-t3ch-wizard** [comment]
@codewarnab is there something more that is required to be done here?

**Comment by codewarnab** [comment]
> @codewarnab is there something more that is required to be done here?

no its ok i will merge soon after reviewing ,i am little busy with  my exams ...

**Comment by codewarnab** [comment]
good job

**Pull request merged**